WordPress/Classes/ViewRelated/Suggestions/SuggestionsTableView.swift

```diff
@@ -97,39 +177,6 @@ extension SuggestionType {
         }
     }
 
-    func moveProminentSuggestionsToTop() {
```

Comment:
I simply moved this method a bit up, and I made it take the `searchResults` as a param instead of accessing `self.searchResults` directly.